examples/default/src/screens/CrashReportingScreen.tsx

```diff
@@ -46,6 +50,48 @@ export const CrashReportingScreen: React.FC = () => {
     }
   }
 
+  const styles = StyleSheet.create({
+    inputWrapper: {
+      padding: 4,
+      flex: 1,
+    },
+
+    formContainer: {
+      flexDirection: 'row',
+      alignItems: 'stretch',
+    },
+  });
+  const [userAttributeKey, setUserAttributeKey] = useState('');
+  const [userAttributeValue, setUserAttributeValue] = useState('');
+  const [crashNameValue, setCrashNameValue] = useState('');
+  const [crashFingerprint, setCrashFingerprint] = useState('');
+  const [crashLevelValue, setCrashLevelValue] = useState<NonFatalErrorType>(
+    NonFatalErrorType.error,
+  );
+
+  function sendCrash() {
+    try {
+      const error = Error(crashNameValue);
+      if (!error.message) {
+        const appName = 'Instabug Test App';
+        error.message = `Handled ${crashNameValue} From ${appName}`;
+      }
+      throw error;
+    } catch (err) {
+      if (err instanceof Error) {
+        const attrMap: { [k: string]: string } = {};
+        attrMap[userAttributeKey] = userAttributeValue;
+
+        CrashReporting.reportError(
+          err,
+          userAttributeValue.length === 0 || userAttributeKey.length === 0 ? null : attrMap,
+          crashFingerprint.length === 0 ? null : crashFingerprint,
+          crashLevelValue,
+        ).then(() => Alert.alert(`Crash report for ${crashNameValue} is Sent!`));
```

Comment:
I think it'd be more readable to use `Record<string, string>` for `userAttributes` and simplify the conditions a bit.

I feel like the API signature is a bit verbose, so hopefully it gets better after the proposed change.

```suggestion
        const userAttributes: Record<string, string> = {};
        if (userAttributesKey && userAttributesValue) {
          userAttributes[userAttributesKey] = userAttributesValue;
        }

        CrashReporting.reportError(
          err,
          userAttributes,
          crashFingerprint || undefined,
          crashLevelValue,
        ).then(() => Alert.alert(`Crash report for ${crashNameValue} is Sent!`));
```